crates/bevy_math/src/curve/mod.rs

```diff
@@ -968,19 +971,135 @@ where
     }
 }
 
+/// Convert a `start` and `end` point of some [`VectorSpace`] type into a [`Curve`] over the
+/// [unit interval], sampled by linearly interpolating between the two boundary values.
+///
+/// [unit interval]: `Interval::UNIT`
+pub fn line_curve<T: VectorSpace>(start: T, end: T) -> impl Curve<T> {
+    FunctionCurve {
+        domain: Interval::UNIT,
+        f: move |t: f32| start * (1.0 - t) + end * t,
+        _phantom: PhantomData,
+    }
+}
+
+/// A [`Curve`] mapping the [unit interval] to itself.
+///
+/// It uses the function `f(x) = x²`
+///
+/// [unit domain]: `Interval::UNIT`
+pub fn quadratic_ease_in() -> impl Curve<f32> {
```

Comment:
I think it might be worth noting why this is desired - there is no sudden jump in velocity at t=0, leading to a smooth speedup.